docs/run_test/platform.rst

```diff
@@ -169,6 +172,11 @@ deployment.
   public IP addresses.  False means to connect with the private IP addresses.
   If not provided, the connections will default to using the public IP
   addresses.
+* **ignored_capability**. Specify feature names which will be ignored in 
+  test requirement. It is useful to ignore some features which are not supported 
+  by the specified VM size. For example, if the VM size is not isolated type, 
+  but you still want to run the test with this size, then you can add 
+  `Isolated_Resource` into this list.
```

Comment:
The key thing is to explain where the feature name is from, so people can know how to get it.